components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -35,6 +35,7 @@ @interface MDCTabBarView ()
 @end
 
 @implementation MDCTabBarView
+@dynamic delegate;
```

Comment:
If the means overriding scrollable view delegate, please adds some comments such like:
// We're overriding UIScrollViewDelegate's delegate solely to change its type (we don't provide
// a getter or setter implementation), thus the @dynamic.

Reply:
Done.